cpp/models/ode_secirvvs/model.h

```diff
@@ -815,7 +817,8 @@ template <typename FP = double, class Base = mio::Simulation<Model<FP>, FP>>
 auto get_migration_factors(const Simulation<Base>& sim, FP /*t*/, const Eigen::Ref<const Vector<FP>>& y)
 
 {
-    auto& params = sim.get_model().parameters;
+    const auto TestAndTraceCapacitySymptomaticMigration = 5.0;
```

Comment:
Can we also add this as model parameter?

Reply:
I had thought about this for a while. I came to the decision to use the actual secirvvs parameter `TestAndTraceCapacityMaxRiskSymptoms`. The previous value of 5.0 looks to me as if the value from the Secirhurd model was incorrectly used here.